app/views/declaration_details.scala.html

```diff
@@ -83,13 +83,25 @@ <h2 class="declaration-details-refs govuk-heading-s">@text</h2>
     <div>
         @link(
             id = Some("view-declaration"),
-            text = messages("submissions.viewDeclaration"),
+            text = messages("submissions.view.declaration"),
             call = routes.SubmissionsController.viewDeclaration(submission.uuid)
         )
     </div>
   }
 }
 
+@copyDeclarationLink = {
+  @if(!isDeclarationRejected(submission)){
```

Comment:
The ticket mentions also not displaying the link if "in Draft mode", which I take to mean checking the `readyForSubmission` flag to be true.

Reply:
Or checking `status` in cache maybe?

Reply:
Yeah I'm just thinking neither of those are required because the declaration has already been submitted :laughing:

Reply:
Indeed :+1: